.github/workflows/integration-darwin.yml

```diff
@@ -8,7 +8,7 @@ permissions: read-all
 jobs:
 
   build:
-    name: PR integration tests
+    name: PR unit and integration tests
```

Comment:
would it be difficult to make these separate jobs? two benefits we get from that:
1) unit/integration tests can run in parallel
2) unit test failure more easily distinguishable from integration test passing, and vice versa

Reply:
I've made this change and split them out.  Initially I thought we might want the unit tests to block the running of an integration test (thinking that if the unit tests failed the integration tests would definitely fail).  I think it makes more sense to run them in parallel though, esp with the integration tests being the time bottleneck.